### PR TITLE
Charts for reconciler

### PR DESCRIPTION
**Pull request by rJankowski93**

### CODE REVIEW

**Review by suleymanakbas91 — APPROVED** [review]

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| `global.database.managedGCP.provisioner.password` | Specifies the password to the user's database in the Runtime Provisioner. | string | "" |
+| `global.database.managedGCP.provisioner.name` | Specifies the database name in the Runtime Provisioner. | string | "" |
+| `global.database.managedGCP.broker.user` | Specifies the database username in Kyma Environment Broker. | string | "" |
+| `global.database.managedGCP.broker.password` | Specifies the password to the user's database in Kyma Environment Broker. | string | "" |
+| `global.database.managedGCP.broker.name` | Specifies the database name in Kyma Environment Broker. | string | "" |
+| `global.database.managedGCP.host` | Specifies cloudsql-proxy host | string | "localhost" |
+| `global.database.managedGCP.hostPort` | Specifies cloudsql-proxy port | string | "5432" |
+| `global.database.managedGCP.sslMode` | Specifies SSL connection mode | string | "" |
+
+To connect to managed database, we use [cloudsql-proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) provided by Google, which consumes `serviceAccountKey` and `instanceConnectionName` values.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:34]:
```suggestion
To connect to the managed database, use [Cloud SQL proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) provided by Google, which consumes the `serviceAccountKey` and `instanceConnectionName` values.
```

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+To find `Instance connection name`, go to the [SQL Instances page](https://console.cloud.google.com/sql/instances) and open desired database overview.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:36]: I'd add this info to the appropriate description in the table.

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+![Instance connection String](./assets/sql-instances-list.png)
+
+Than look for `Instance connection name` box inside `Connect to this instance` section.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:39]: I'd add this info to the appropriate description in the table.

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+![Instance connection String](./assets/instance-connection-string.png)
+
+For the `director.user`, `provisioner.user`, `broker.user`, `director.password`, `provisioner.password`, and `broker.password` values, use one of the accounts from the **USERS** tab. The `director.name`, `provisioner.name`, and `broker.name` values are the names of available databases. For the list of available names, see the **DATABASES** tab.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:43]:
```suggestion
>**NOTE:** For the `director.user`, `provisioner.user`, `broker.user`, `director.password`, `provisioner.password`, and `broker.password` values, use one of the accounts from the **USERS** tab. The `director.name`, `provisioner.name`, and `broker.name` values are the names of available databases. For the list of available names, see the **DATABASES** tab.
```

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+The `host` and the `hostPort` values specifies the cloudsql-proxy host and port. These are used directly by application to connect to proxy, and further to database.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:45]: I'd add this info to the appropriate description in the table.

File: resources/reconciler/configure-managed-gcp-postgresql.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+The `sslMode` value specifies SSL connection mode to database. Check possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html) under `SSL Mode Descriptions` section.
```
Comment (klaudiagrz) [inline resources/reconciler/configure-managed-gcp-postgresql.md:47]: I'd add this info to the appropriate description in the table.

**Review by klaudiagrz — APPROVED** [review]

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**